scripts/deploy/nlp/query_trtllm_pytorch.py

```diff
@@ -0,0 +1,98 @@
+    )
+    parser.add_argument("-u", "--url", default="0.0.0.0", type=str, help="url for the triton server")
+    parser.add_argument("-mn", "--model_name", required=True, type=str, help="Name of the triton model")
+    prompt_group = parser.add_mutually_exclusive_group(required=True)
+    prompt_group.add_argument("-p", "--prompt", required=False, type=str, help="Prompt")
+    prompt_group.add_argument("-pf", "--prompt_file", required=False, type=str, help="File to read the prompt from")
+    parser.add_argument("-ml", "--max_length", default=256, type=int, help="Max output token length")
+    parser.add_argument("-tk", "--top_k", type=int, help="top_k")
+    parser.add_argument("-tpp", "--top_p", type=float, help="top_p")
+    parser.add_argument("-t", "--temperature", type=float, help="temperature")
+    parser.add_argument("-it", "--init_timeout", default=60.0, type=float, help="init timeout for the triton server")
+
+    args = parser.parse_args(argv)
+    return args
+
+
+def query_llm(
+    url,
+    model_name,
+    prompts,
+    max_length=256,
+    top_k=None,
+    top_p=None,
+    temperature=None,
+    init_timeout=60.0,
+):
+    """Query a TensorRT-LLM PyTorch backend model deployed on Triton Inference Server.
+
+    Args:
+        url (str): URL of the Triton Inference Server (e.g. localhost or IP address)
+        model_name (str): Name of the model as deployed on Triton server
+        prompts (List[str]): List of text prompts to send to the model
+        max_length (int, optional): Maximum number of tokens to generate in the response. Defaults to 256.
+        top_k (int, optional): Number of highest probability tokens to consider for sampling. Defaults to None.
+        top_p (float, optional): Cumulative probability threshold for token sampling. Defaults to None.
+        temperature (float, optional): Temperature for controlling randomness in sampling (higher = more random). Defaults to None.
+        init_timeout (float, optional): Timeout in seconds when initializing connection to Triton server. Defaults to 60.0.
+
+    Returns:
+        List[str]: Generated text responses for each input prompt
+    """
+    nemo_query = NemoQueryTRTLLMPytorch(url, model_name)
+    return nemo_query.query_llm(
+        prompts=prompts,
+        max_length=max_length,
+        top_k=top_k,
+        top_p=top_p,
+        temperature=temperature,
+        init_timeout=init_timeout,
+    )
+
+
+def query(argv):
+    args = get_args(argv)
+
+    if args.prompt_file is not None:
+        with open(args.prompt_file, "r") as f:
+            args.prompt = f.read()
+
+    outputs = query_llm(
+        url=args.url,
+        model_name=args.model_name,
+        prompts=[args.prompt],
+        max_length=args.max_length,
+        top_k=args.top_k,
+        top_p=args.top_p,
+        temperature=args.temperature,
+        init_timeout=args.init_timeout,
+    )
+    print(outputs)
+
+
+if __name__ == '__main__':
+    query(sys.argv[1:])
```

Comment:
NIT no need to pass `sys.argv[1:]` around, `query()` should work (after removing `argv` from `get_args` etc.)